packages/vm/src/cache.rs

```diff
@@ -217,8 +217,8 @@ where
         let module = compile_and_use(&wasm, Some(self.instance_memory_limit))?;
         let instance =
             Instance::from_module(&module, backend, options.gas_limit, options.print_debug)?;
-        self.fs_cache.store(checksum, &module)?;
-        self.memory_cache.store(checksum, module)?;
+        let module_size = self.fs_cache.store(checksum, &module)?;
+        self.memory_cache.store(checksum, module, module_size)?;
```

Comment:
This now looks as if the size in memory is the same as the serialized size. It is a good approximation, but not the exact weight. However, I don't know how to best express this in code.

Reply:
The approximation is good enough, than that's not necessary.

We can add a comment somewhere, probably inside these methods, detailing that the serialized module size is a good approximation for the in-memory module size, and providing some values.